src/markdown-pages/docs/managers/github-manager.md

```diff
@@ -0,0 +1,290 @@
+
+
+# <a name="overview"></a>Overview
+
+
+If there is an open issue in a GitHub repository that tracks a known problem which is expected to cause a test class or test method to fail, you can add the _@GitHubIssue_ annotation to a test class or test method so that the expected failure is recorded as _Failed with Defects_  in the test report. 
+
+Using this annotation on a test class or test method means that the expected failure is clearly identifiable in the test report, making it easier to spot any unexpected test failures and saving time by avoiding the investigation of a known issue. 
+
+
+## Using the Manager
+
+The GitHub Manager is activated if there is a _@GitHubIssue_ annotation on the test class or any of its test methods. The presence of the _@GitHubIssue_ annotation means that there is a known problem which is expected to affect the result of the test. 
+
+If the issue is expected to affect one or more specific test methods, add the annotation on the test method. If the scope of the issue is larger and might affect the whole test class, add the annotation at the class level.
+
+To use the GitHub Manager you must import the _@GitHubIssue_ annotation into the test, as shown in the following example, which is based on the _BatchAccountsOpenTest_ that is provided with Galasa SimBank: 
+
+```
+import dev.galasa.githubissue.GitHubIssue;
+...
+@Test
+@GitHubIssue(issue = "1000", repo = "galasa-dev/projectmanagement")
+public class BatchAccountsOpenTest {
+```
+
+Use the following examples, which are based on _BatchAccountsOpenTest_ to understand how to add the _@GitHubIssue_ annotation for GitHub issue number _1000_ to a test method and to a test class. 
+
+### Example
+
+GitHub issue _1000_ has a status of _open_ on GitHub in the _galasa-dev/projectmanagement_ repository. Issue _1000_ tracks a known problem with the behaviour of the 
+Z/OS Batch Manager. The issue causes the test class _BatchAccountsOpenTest_ to fail.
+
+The following example adds the _@GitHubIssue_ annotation on the test method:
+
+```
+@Test
+@GitHubIssue(issue = "1000", repo = "galasa-dev/projectmanagement")
+public void batchOpenAccountsTest()
+```
+
+The following example adds the _@GitHubIssue_ annotation on the test class:
+
+```
+@Test
+@GitHubIssue(issue = "1000", repo = "galasa-dev/projectmanagement")
+public class BatchAccountsOpenTest
+```
+
+Once the GitHub issue is closed, remove the _@GitHubIssue_ annotation from the test class or test method.
+
+
+### Using Regex
+
+To ensure that the failing exception is due to the known problem that is identified in the GitHub issue, use Regex in the annotation to check that the recorded test failure matches the string that is specified in Regex. If the failing exception does not match the Regex, the test has failed for a different or unknown reason, not the reason that is specified in the GitHub issue. If no Regex is provided in the annotation, any failing exception is accepted.
+
```

Comment:
we should show what text from the failure is matched by the regex

Reply:
I have been meaning to provide Caroline with an example of a failure and the corresponding Regex - on my to do list

Reply:
sent a suggestion over for review